styles.css

```diff
@@ -89,23 +108,29 @@ label {
   border: 2px solid;
```

Comment:
what if we try out a display: grid? Maybe we can get these down to the right size better than with flex since it is wrapping them all right now